CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ FEATURES:
   * **New resource: `google_compute_project_metadata`** [GH-3065]
   * **New resources: `aws_ami`, `aws_ami_copy`, `aws_ami_from_instance`** [GH-2874]
   * **New resource: `google_storage_bucket_object`** [GH-3192]
+  * **New resources: `google_compute_vpn_gateway`, `google_compute_vpn_tunnel`** [GH-3213]
```

Comment:
FYI: The changelog is something maintainers update via github web iface when they're merging the PR to avoid any potential conflicts.